src/pages/workspaces/workspace/Dashboard.js

```diff
@@ -270,6 +267,50 @@ const WorkspaceDashboard = _.flow(
     refresh()
   })
 
+  const getCloudInformation = () => {
+    return !googleProject && !azureContext ? [] : [
+      dl({},
+        !!googleProject ? [
+          h(InfoRow, { title: 'Cloud Name' }, [
+            h(GcpLogo, { title: 'Google Cloud Platform', role: 'img', style: { height: 16 } })
```

Comment:
@MatthewBemis I removed `width: 132, marginLeft: -15` here and it looks fine. Do you recall why you had those values?

Reply:
I cannot recall. All I know is that I am still getting my feet wet when it comes to styling. I tried to go back to clean up as many of the oddities as possible but I must have missed this one. Thanks for cleaning it up!